src/modules/Elsa.Quartz/Services/JobKeyProvider.cs

```diff
@@ -0,0 +1,19 @@
+using Elsa.Common.Multitenancy;
+using Elsa.Quartz.Contracts;
+using Quartz;
+
+namespace Elsa.Quartz.Services;
+
+internal class JobKeyProvider(ITenantAccessor tenantAccessor) : IJobKeyProvider
+{
+    public JobKey GetJobKey<TJob>() where TJob : IJob
+    {
+        return new(typeof(TJob).Name, GetGroupName());
```

Comment:
Will `typeof(TJob).Name` generate a unique name or will this be the same for each workflowDefinition? If not, should we not use something like the instanceId here instead?